tests/js/d3ViewComponentTests.js

```diff
@@ -0,0 +1,163 @@
+/*!
+Copyright 2015 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/gpii/universal/LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.registerNamespace("gpii.tests");
+
+    fluid.defaults("gpii.tests.d3ViewComponent", {
+        gradeNames: ["gpii.d3ViewComponent", "autoInit"],
+        listeners: {
+            "onCreate.addMouseoverListener": {
+                listener: "{that}.addD3Listeners",
+                args: ["{that}.container", "mouseover", "gpii.tests.mouseOverListener"]
+            }
+        },
+        members: {
+            mouseOverListenerCalled: false
+        }
+    });
+
+    gpii.tests.mouseOverListener = function (data, i, that) {
+        that.mouseOverListenerCalled = true;
+    };
+
+    jqUnit.test("Test d3ViewComponent API", function () {
+        jqUnit.expect(3);
+
+        var that = gpii.tests.d3ViewComponent(".gpiic-d3");
+
+        // The D3 DOM event listener is registered
+        jqUnit.assertFalse("The mouseover listener for pie slices have not been triggered", that.mouseOverListenerCalled);
+
+        var d3Elem = that.jQueryToD3(that.container);
+        jqUnit.assertEquals("The jQuery element has been converted to D3 element", that.container[0], d3Elem[0][0]);
```

Comment:
Agree. However, there is no requirement to have a D3 to jQuery conversion with the implementation so far. Figure we don't wanna leave an isolated function and forget it later on. But I'm open to write that function if you think it's better to have the counterpart in now.